atomic_reactor/tasks/binary.py

```diff
@@ -6,10 +6,21 @@
 of the BSD license. See the LICENSE file for details.
 """
 
+from atomic_reactor.dirs import RootBuildDir
 from atomic_reactor.tasks import plugin_based
 
 
-class BinaryPreBuildTask(plugin_based.PluginBasedTask):
+class BinaryContainerBuildTask(plugin_based.PluginBasedTask):
```

Comment:
Maybe just `BinaryTask` or `BinaryContainerTask`? I think the current name is too similar to `BinaryBuildTask`

Reply:
yeah this is very confusing

since this is class used for other classes, maybe even something like BinaryGeneralTask .....